apps/api/src/app/inbox/usecases/notifications-count/notifications-count.command.ts

```diff
@@ -0,0 +1,13 @@
+import { IsArray, IsOptional } from 'class-validator';
+import type { MessagesStatusEnum } from '@novu/shared';
+
+import { EnvironmentWithSubscriber } from '../../../shared/commands/project.command';
+
+export class NotificationsCountCommand extends EnvironmentWithSubscriber {
+  @IsOptional()
+  @IsArray()
+  readonly feedId?: string[];
```

Comment:
`FeedId`? Isn't this replaced with filters?

Reply:
I forgot to remove that, it's not used ;)